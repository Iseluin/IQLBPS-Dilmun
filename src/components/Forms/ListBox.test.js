import renderer from 'react-test-renderer';
import { ListBox } from '.'

const options = [
    'test',
    'test1',
    'test2',
    'test3',
];

test('ListBox renders correctly', () => {
    const tree = renderer.create(<ListBox options={options} />).toJSON();
    expect(tree).toMatchSnapshot();
});
