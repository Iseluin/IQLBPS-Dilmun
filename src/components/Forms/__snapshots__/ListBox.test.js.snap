// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListBox renders correctly 1`] = `
<div
  className="w-full mt-4"
>
  <div
    className="relative mt-1"
  >
    <button
      aria-expanded={false}
      aria-haspopup={true}
      className="relative w-full py-2 pl-3 pr-10 text-left bg-background  text-primary font-medium rounded-lg shadow-md cursor-default focus:outline-none focus-visible:ring-2 focus-visible:ring-opacity-75 focus-visible:ring-white focus-visible:ring-offset-orange-300 focus-visible:ring-offset-2 focus-visible:border-indigo-500 sm:text-sm"
      data-headlessui-state=""
      disabled={false}
      id="headlessui-listbox-button-:r0:"
      onClick={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      type="button"
    >
      <span
        className="block truncate"
      />
      <span
        className="absolute inset-y-0 right-0 flex items-center pr-2 pointer-events-none"
      >
        <svg
          aria-hidden="true"
          className="w-5 h-5 text-gray-400"
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          strokeWidth="0"
          style={
            Object {
              "color": undefined,
            }
          }
          viewBox="0 0 20 20"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            clipRule="evenodd"
            d="M10 3a1 1 0 01.707.293l3 3a1 1 0 01-1.414 1.414L10 5.414 7.707 7.707a1 1 0 01-1.414-1.414l3-3A1 1 0 0110 3zm-3.707 9.293a1 1 0 011.414 0L10 14.586l2.293-2.293a1 1 0 011.414 1.414l-3 3a1 1 0 01-1.414 0l-3-3a1 1 0 010-1.414z"
            fillRule="evenodd"
          />
        </svg>
      </span>
    </button>
  </div>
</div>
`;
